k8s/templates/values/values.yml

```diff
@@ -0,0 +1,7 @@
+#@data/values
+---
+version: "74.12.0"
```

Comment:
It's worth noting we will need to manage making sure this version label is kept updated in CI

Reply:
Yes, we certainly will.